### DISCUSSION TITLE
子图标题与子图之间的间距过大

### DISCUSSION THREAD

**Discussion by severous** [note286/xduts#52] [Q&A] [answered]
子图标题与子图之间的间距相较于word的单倍行距过大。
使用的代码如下所示：
```latex
\begin{figure}[ht]
 \centering
 \subfloat[]{\includegraphics[width=2.8cm]{source/1.jpg}} 
 \subfloat[]{\includegraphics[width=2.8cm]{source/1.jpg}}
 \subfloat[]{\includegraphics[width=2.8cm]{source/1.jpg}}
 \\
 \subfloat[]{\includegraphics[width=2.8cm]{source/1.jpg}} 
 \subfloat[]{\includegraphics[width=2.8cm]{source/1.jpg}}
 \subfloat[]{\includegraphics[width=2.8cm]{source/1.jpg}}
 \caption{小狗图} 
 \label{fig.work1_fig10}
\end{figure}
```
latex生成的图片布局如下所示：

![14531677382710_ pic](https://user-images.githubusercontent.com/37959509/221391080-a7f285e2-c121-4d91-be7d-6a107f9b0ef3.jpg)
word中的图片布局如下所示
![14541677382738_ pic](https://user-images.githubusercontent.com/37959509/221391091-35a3137f-ea4d-4743-9f43-fc5e55e7fd1a.jpg)
[文档中提及的命令`style/caption-label-sep`似乎不适用于子图,根据[此链接](https://blog.csdn.net/weixin_43849277/article/details/115201445)计算的单倍行距为13.64pt(5号字体),设置后似乎不起作用

**Comment by note286** [comment, marked as answer]
首先，没必要刻意模仿Microsoft Word。

其次，`style/caption-label-sep`是设置`caption`与中标签与标题内容之间的水平间距的，例如`图1`和`小狗图`之间的间距。我觉得我手册中对此的描述应该没有歧义。

此外，我在 #40 中给了示例，推荐使用`subcaption`排版子图，更多设置可以阅读`subcaption`手册，例如`\captionsetup[subfigure]{skip=0pt}`的用法就在7.4节有介绍。

![image](https://user-images.githubusercontent.com/16207292/221391693-4d93ba5d-68d8-4a7d-9c09-e15d3a75f6c1.png)


```latex
\documentclass{xdupgthesis}
\usepackage{subcaption}
\captionsetup[subfigure]{skip=0pt}
\begin{document}
\begin{figure}
\centering
\subcaptionbox{}{\includegraphics[width=.3\linewidth]{example-image}}\hfill
\subcaptionbox{}{\includegraphics[width=.3\linewidth]{example-image}}\hfill
\subcaptionbox{}{\includegraphics[width=.3\linewidth]{example-image}}\hfill
\subcaptionbox{}{\includegraphics[width=.3\linewidth]{example-image}}\hfill
\subcaptionbox{}{\includegraphics[width=.3\linewidth]{example-image}}\hfill
\subcaptionbox{}{\includegraphics[width=.3\linewidth]{example-image}}
\caption{小狗图}
\end{figure}
\end{document}
```

![image](https://user-images.githubusercontent.com/16207292/221391632-2226df14-54e0-42f2-90c9-f641d3d93a2c.png)

最后，强烈不建议用户手动指定浮动体的位置，例如你的示例中的`[ht]`。
